lib/compat/wordpress-6.0/class-wp-theme-json-gutenberg.php

```diff
@@ -30,6 +71,98 @@ class WP_Theme_JSON_Gutenberg extends WP_Theme_JSON_5_9 {
 		'version',
 	);
 
+	/**
+	 * The valid properties under the settings key.
+	 *
+	 * @var array
+	 */
+	const VALID_SETTINGS = array(
+		'appearanceTools' => null,
+		'border'          => array(
+			'color'  => null,
+			'radius' => null,
+			'style'  => null,
+			'width'  => null,
+		),
+		'color'           => array(
+			'background'       => null,
+			'custom'           => null,
+			'customDuotone'    => null,
+			'customGradient'   => null,
+			'defaultGradients' => null,
+			'defaultPalette'   => null,
+			'duotone'          => null,
+			'gradients'        => null,
+			'link'             => null,
+			'palette'          => null,
+			'text'             => null,
+		),
+		'custom'          => null,
+		'layout'          => array(
+			'contentSize' => null,
+			'position'    => null,
```

Comment:
Yes, I'm pretty sure it will be! There's still a few other things I wanted to explore in this PR before adding that (like where to put `top`, `right`, `bottom`, `left` in the object, and whether we also need to store a `zIndex` value so that the sticky position will work — even if we keep that setting hidden from the user in the UI)